java-manta-client-unshaded/src/test/java/com/joyent/manta/client/multipart/ServerSideMultipartManagerTest.java

```diff
@@ -344,6 +344,72 @@ public void willFailToCreateCommitRequestBodyWhenThereAreNoParts() {
         ServerSideMultipartManager.createCommitRequestBody(Stream.empty());
     }
 
+    public void canExtrapolatePrefixDirectoryForObjectIdForOneToEight() {
+        final String uuidBase = "0077e0b8-fb7d-eddd-b106-8e8e3ca3d8b";
```

Comment:
Would we wanted to use a random-UUID generator to create a UUID base and then use it for all our use-cases that fail by appending the last char's/ digit values on which it fails instead of hard-coding all the input values ?

Reply:
I had considered that approach. However, this approach allows us to have a deterministic test. Each execution will be exactly the same in the current approach which makes it easier to understand.